modules/ballerina-http/src/main/ballerina/ballerina/net/http/connection.bal

```diff
@@ -0,0 +1,11 @@
+package ballerina.net.http;
+
+@Description { value:"Sends a 100-continue response to the client."}
+@Param { value:"conn: The server connector connection" }
```

Comment:
I feel that this description may not be user friendly. Will it be possible to improve this? Is the user likely to understand what we mean when we just simply say "The server connector connection"?

Reply:
Also, isn't there any other place we can put this function?

Reply:
keeping the same description as user can get the rough idea about the conn with it.